### ISSUE TITLE
server failing to start

### ISSUE THREAD

**Issue by cchall** [radiasoft/jupyter.radiasoft.org#134]
I am getting an error when I try to restart my jupyter server:
Spawn failed: Server at http://10.1.2.13:7924/jupyter/user/cchall/ didn't respond in 45 seconds

**Comment by e-carlin** [comment]
I restarted it. Seems to be up now. Can you get to it? Not sure what the root cause was...

**Comment by cchall** [comment]
Initially when I tried to go to it I was given the server restarting message which resulted in an error again.
Logging out and back in seemed to fix this and I was taken to a running server.

I know there was some trouble a while ago with my pre_jupyter_bashrc. I had just uncommented part of that when I first tried restarting this morning (after testing to make sure no obvious errors appeared when I ran pre_jupyter_bashrc). Not sure if this has anything to do with it or if it's a completely different problem now.
